src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -2408,7 +2409,7 @@ def load_arguments(self, _):
                    help='Name of resource group.')
         c.argument('cluster_name',
                    options_list=['--cluster-name', '-c'],
-                   validator=validate_location_cluster_name_resource_group_mutually_exclusive,
+                   validator=validate_location_cluster_parameters_present,
```

Comment:
Don't you also want to validate that location and cluster name are mutually exclusive?

maybe your validation method needs to do both?